app/controllers/hq/partners_controller.rb

```diff
@@ -6,6 +6,19 @@ def index
     @partners= Partner.all
   end
 
+  def new
+    @partner = Partner.new
+    load_associations
+  end
+  
+  def create 
+    build_partner
+    unless save_partner
```

Comment:
the load_associations could be an else in the save_partner method and then this guard could be removed from create and I could lose the call to load_associations in the update method.